YoutubeDownloader/Views/Components/DashboardView.xaml

```diff
@@ -11,6 +11,11 @@
     d:DataContext="{d:DesignInstance Type=components:DashboardViewModel}"
     FocusManager.FocusedElement="{Binding ElementName=QueryTextBox}"
     mc:Ignorable="d">
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <s:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" FalseVisibility="Hidden" />
+        </ResourceDictionary>
```

Comment:
Added a comment to explain the need for this, please check if this sounds right.

```suggestion
        <!-- DataGrid's autosizing breaks when it's filled with rows while it's collapsed (rather than hidden) -->
        <ResourceDictionary>
            <s:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" FalseVisibility="Hidden" />
        </ResourceDictionary>
```